test/test_terraform_controller.py

```diff
@@ -39,12 +41,12 @@ def test_get_instances(self, mocker, tf_controller, cloud):
         mock_read = mocker.MagicMock(return_value='test_json')
         os._wrap_close.read = mock_read
 
-        mock_loads = mocker.patch(
-            'json.loads',
-            return_value={'values': {'root_module': {'resources': 'test'}}},
-        )
+        mock_loads = mocker.patch('json.loads',
+                                  return_value={'values': {'root_module': {'resources': 'test'}}})
 
-        mock_get_instances_cloud = mocker.MagicMock(return_value={'test_dict': 42})
+        test_dict = {'test_dict': 42}
+        mock_get_instances_cloud = mocker.MagicMock(
```

Comment:
I think the code linter messed it up here. Same at line 31, 90, 98.

Reply:
This identation follows pep8 standard, but i looked weird as I didn't had the max-line-lengh exception to 120

Reply:
I see!